exotica_core/src/dynamics_solver.cpp

```diff
@@ -167,4 +167,10 @@ void AbstractDynamicsSolver<T, NX, NU>::SetIntegrator(std::string integrator_in)
         ThrowPretty("Unknown integrator: " << integrator_in);
 }
 
+template <typename T, int NX, int NU>
+Eigen::Matrix<T, NX, 1> AbstractDynamicsSolver<T, NX, NU>::StateDelta(const StateVector& x_1, const StateVector& x_2)
```

Comment:
Should we perhaps declare this `inline`?

Reply:
How would overloading work when it becomes more complicated for derived classes with `inline`?

Reply:
It can theoretically work: https://stackoverflow.com/questions/733737/are-inline-virtual-functions-really-a-non-sense#733791 - but let's leave as is.

Reply:
Haha. I was just about to paste this here. OK, I'll change it.